packages/analysis-report/tests/__snapshots__/SiteSEOReportTest.js.snap

```diff
@@ -10,78 +10,78 @@ exports[`The SiteSEOReport testing suite The SiteSEOReport should match the snap
     Test assessment
   </p>
   <div
-    className="bjXoTs progress"
+    className="bDYRu progress"
```

Comment:
I am not sure why the test in this file failed when I didn't touch anything in `analysis-report` package. Looking at the updated snapshot which is only about updating the random keys, I think that this is something that we don't need to worry about